core/src/test/java/org/apache/iceberg/io/FileIOTrackerTestUtil.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.iceberg.io;
+
+import com.github.benmanes.caffeine.cache.Cache;
+import org.apache.iceberg.TableOperations;
+
+public abstract class FileIOTrackerTestUtil {
```

Comment:
Why is this an abstract class?

Util classes are usually normal classes with a private parameterless constructor

Reply:
This new class is part of the dependency PR. Should we discuss it [there](https://github.com/apache/iceberg/pull/14378)?